.github/workflows/deploy_to_aws.yml

```diff
@@ -261,6 +261,43 @@ jobs:
           string: ${{ vars.ENV_NAME }}
       - name: Checkout repo
         uses: actions/checkout@v4
+      - name: Stop Logstash for ${{ vars.ENV_NAME }}
+        if: ${{ (vars.ENV_NAME != 'Production' && vars.ENV_NAME != 'Staging' && vars.ENV_NAME) && inputs.restore-db == true}}
+        run: |
+          docker run --rm \
+              -e AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }} \
+              -e AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }} \
+              -e AWS_DEFAULT_REGION=eu-west-1 \
+              amazon/aws-cli \
+              ecs update-service --desired-count 0 --cluster=ecsOpenSupplyHub${{vars.ENV_NAME}}Cluster \
+              --service=OpenSupplyHub${{vars.ENV_NAME}}AppLogstash  --region=eu-west-1
+      - name: Clear OpenSearch' index for ${{ vars.ENV_NAME }}
+        if: ${{ (vars.ENV_NAME != 'Production' && vars.ENV_NAME != 'Staging' && vars.ENV_NAME) && inputs.restore-db == true}}
+        run: |
+          mkdir -p script
+          mkdir -p ssh
+          echo -e "Host *\n\tStrictHostKeyChecking no\n\n" > ssh/config
+          printf "%s\n" "${{ secrets.SSH_PRIVATE_KEY }}" > ssh/id_rsa
+          echo "" >> ssh/id_rsa
+          echo -n ${{ vars.BASTION_IP }} > script/.env
+          cat <<EOF > script/clear_opensearch.sh
+          #!/bin/bash
+          curl -X DELETE https://${{ vars.OPENSEARCH_DOMAIN }}/production-locations --aws-sigv4 "aws:amz:eu-west-1:es" --user "${{ secrets.AWS_ACCESS_KEY_ID }}:${{ secrets.AWS_SECRET_ACCESS_KEY }}"
+          sudo mount -t efs -o tls,accesspoint=${{ vars.EFS_AP_ID }} ${{ vars.EFS_ID }}:/ /mnt
+          sudo rm /mnt/logstash_jdbc_last_run
+          sudo umount /mnt
+          EOF
+          cat <<EOF > script/run.sh
+          chmod 700 /root/.ssh
+          chmod 400 /root/.ssh/id_rsa
+          chmod +x /script/clear_opensearch.sh
+          scp /script/clear_opensearch.sh ec2-user@${{ vars.BASTION_IP }}:
+          ssh ec2-user@${{ vars.BASTION_IP }} ./clear_opensearch.sh
+          EOF
+          docker run --rm  \
+              -v ./script:/script \
+              -v ./ssh:/root/.ssh \
+              kroniak/ssh-client bash /script/run.sh
```

Comment:
Is it possible to place these scripts inside the [deployment](https://github.com/opensupplyhub/open-supply-hub/tree/main/deployment) folder? For instance, like run_kafka_task bash script and others.

Reply:
No. Because the scripts will be running on a remote host and won't be able to use GitHub Runner environment variables.